### PR TITLE
Consistent syntax postgres-create-schema.sql

### PR DESCRIPTION
**Pull request by ignasi35**

### CODE REVIEW

File: core/src/main/resources/schema/postgres/postgres-create-schema.sql [hunk 1]
```diff
@@ -47,16 +47,16 @@ CREATE TABLE IF NOT EXISTS public.snapshot (
...
-    "global_offset" BIGSERIAL,
-    "persistence_id" VARCHAR(255) NOT NULL,
-    "revision" BIGINT NOT NULL,
-    "state_payload" BYTEA NOT NULL,
-    "state_serial_id" INTEGER NOT NULL,
-    "state_serial_manifest" VARCHAR(255),
-    "tag" VARCHAR,
-    "state_timestamp" BIGINT NOT NULL,
-    PRIMARY KEY("persistence_id")
+CREATE TABLE IF NOT EXISTS public.state (
```
Comment (ignasi35) [inline core/src/main/resources/schema/postgres/postgres-create-schema.sql:50]: prefix the resources and don't quote table and column names.

**Comment by debasishg** [comment]
@ignasi - In that case will u also please change `postgres-create-schema-legacy.sql` - I noticed that in CI all tests are run against this schema as well.

**Comment by debasishg** [comment]
I remember Patrik mentioned that some of the JDBC tests are flaky as they have too less timeouts that fail on the CI ..

**Comment by ignasi35** [comment]
> @ignasi - In that case will u also please change `postgres-create-schema-legacy.sql` - I noticed that in CI all tests are run against this schema as well.

I didn't realize the legacy schema also supported durable state. Changing...

**Comment by ignasi35** [comment]
IDEA (to be done on a separate PR): maybe rename the table from `state` to `durable_state` for consistency.

**Review by debasishg — APPROVED** [review]
LGTM

**Review by RayRoestenburg — APPROVED** [review]

**Pull request merged**